README.md

```diff
@@ -5,6 +5,10 @@
 </p>
 <br/>
 
+## Overview
+
+The Babylon Wallet Connector provides a unified interface for integrating both Bitcoin (BTC) and Babylon (BBN) wallets into Babylon dApp. It supports both native wallet extensions and injectable mobile wallets.
```

Comment:
Okay

Reply:
Added a paragraph with the difference between the two. I won't put the screenshots right now since these screenshots are most likely to change. But this can be done later on for sure